adb_graphics/datahandler/gribdata.py

```diff
@@ -530,11 +538,11 @@ def fire_weather_index(self, values, **kwargs) -> np.ndarray:
 
         # Gather fields from the input
         veg = values # Chose this value as the main one in the default_specs
-        temp = self.values(name='temp', level='2m', do_transform=False).values
-        dewpt = self.values(name='dewp', level='2m', do_transform=False).values
-        weasd = self.values(name='weasd', level='sfc', do_transform=False).values
-        gust = self.values(name='gust', level='10m', do_transform=False).values
-        soilm = self.values(name='soilm', level='sfc', do_transform=False).values
+        temp = self.values(name='temp', level='2m', do_transform=False)
+        dewpt = self.values(name='dewp', level='2m', do_transform=False)
+        weasd = self.values(name='weasd', level='sfc', do_transform=False)
+        gust = self.values(name='gust', level='10m', do_transform=False)
+        soilm = self.values(name='soilm', level='sfc', do_transform=False)
```

Comment:
Since all are now Xarrays, we shouldn't need to use the "values" method to get the actually data.